code/datums/beam.dm

```diff
@@ -159,19 +159,7 @@
 /obj/effect/ebeam/singularity_act()
 	return
 
-/**
- * This is what you use to start a beam. Example: origin.Beam(target, args). **Store the return of this proc if you don't set maxdist or time, you need it to delete the beam.**
- *
- * Unless you're making a custom beam effect (see the beam_type argument), you won't actually have to mess with any other procs. Make sure you store the return of this Proc, you'll need it
- * to kill the beam.
- * **Arguments:**
- * BeamTarget: Where you're beaming from. Where do you get origin? You didn't read the docs, fuck you.
- * icon_state: What the beam's icon_state is. The datum effect isn't the ebeam object, it doesn't hold any icon and isn't type dependent.
- * icon: What the beam's icon file is. Don't change this, man. All beam icons should be in beam.dmi anyways.
- * maxdistance: how far the beam will go before stopping itself. Used mainly for two things: preventing lag if the beam may go in that direction and setting a range to abilities that use beams.
- * beam_type: The type of your custom beam. This is for adding other wacky stuff for your beam only. Most likely, you won't (and shouldn't) change it.
- */
-/atom/proc/Beam(atom/BeamTarget,icon_state="b_beam",icon='icons/effects/beam.dmi',time=INFINITY,maxdistance=INFINITY,beam_type=/obj/effect/ebeam, beam_color = null, beam_type=/obj/effect/ebeam,beam_sleep_time = 3)
-	var/datum/beam/newbeam = new(src,BeamTarget,icon,icon_state,time,maxdistance,beam_type, beam_color,beam_sleep_time)
+/atom/proc/Beam(atom/BeamTarget,icon_state="b_beam",icon='icons/effects/beam.dmi',time=50, maxdistance=10,beam_type=/obj/effect/ebeam,beam_sleep_time = 3)
+	var/datum/beam/newbeam = new(src,BeamTarget,icon,icon_state,time,maxdistance,beam_type,beam_sleep_time)
```

Comment:
You deleted a large amount of documentation.
Also, if this is related to your beam refactor it's not going to work.
I had already reported a major issue with it that was never fixed

Reply:
reverted, no idea how this got in here my main guess is github desktop pulling changes from an old branch